h/nipsa/subscribers.py

```diff
@@ -1,18 +1,12 @@
-def transform_annotation(event):
-    """Add a {"nipsa": True} field on moderated annotations or those whose users are flagged."""
-    payload = event.annotation_dict
-
-    nipsa = _user_nipsa(event.request, payload)
-
-    if nipsa:
-        payload["nipsa"] = True
-
-
-def _user_nipsa(request, payload):
-    nipsa_service = request.find_service(name="nipsa")
-    return "user" in payload and nipsa_service.is_flagged(payload["user"])
```

Comment:
This stuff was in-lined, allowing an "optimisation" where we don't get the service if we aren't going to call it. Mostly to make the logic clearer.